Bugzilla/Constants.pm

```diff
@@ -740,7 +740,7 @@ sub DEFAULT_CSP {
       'https://crash-stop-addon.herokuapp.com',
     ],
     worker_src  => ['none',],
-    img_src     => ['self', 'blob:', 'https://secure.gravatar.com'],
+    img_src     => ['self', 'data:', 'blob:', 'https://secure.gravatar.com'],
```

Comment:
is data: safe for an img-src?

Reply:
I believe so... [SVG used as image](https://developer.mozilla.org/en-US/docs/Web/SVG/SVG_as_an_Image) cannot execute JavaScript, and MIME types other than `image/*` are simply ignored.

* https://security.stackexchange.com/questions/177412/restrict-js-in-svgs
* https://security.stackexchange.com/questions/94993/is-including-the-data-scheme-in-your-content-security-policy-safe